server/index.js

```diff
@@ -20,7 +20,9 @@ if (process.env.NODE_ENV === 'development') {
 
 app.use(express.static(publicPath));
 
-app.get('/api/users', (req, res, next) => {
+app.get('/api/userList/:currentUser', (req, res, next) => {
+  const { currentUser } = req.params;
```

Comment:
Instead of using `req.params`, just hard-code the current user for now.

```js
app.get('/api/userList', (req, res, next) => {
  const currentUser = 5;
```